src/multipart.ts

```diff
@@ -43,7 +48,7 @@ const multipartDataGenerator = (
     if (Object.prototype.hasOwnProperty.call(v, 'data')) {
       const typedEntry: {
         name: string;
-        data: BufferedFile;
+        data: string | Uint8Array;
```

Comment:
nit: i think `BufferedFile` was correct here.

Reply:
Based on log statements, it seems like `typedEntry.data` gets converted to `string | Uint8Array` because it comes from `v` , which is the result of `utils.flattenAndStringify(data)`.